Bottom: 810627dc78df9ae6549b9d92b58414c164959985
Top:    04f5649bf5734094f8f3e1d4fe0fc9649f7e7bf0
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-07-15 14:05:34 -0400

copy


---

diff --git a/cop b/cop
new file mode 100755
index 0000000..42d5878
--- /dev/null
+++ b/cop
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+
+make zImage uImage dtbs
+#cp /work/hans/sound/soc/sunxi/sunxi-codec.ko /work/root/lib/modules/3.15.0+/kernel/sound/soc/sunxi
+
